### PR TITLE
LPS-196643 Created User profile and menu

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LPS-189188](https://liferay.atlassian.net/browse/LPS-189188)

### CODE REVIEW

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
Good job on that @samuelleolopes , @Mylena-Rodrigues , @Aline160 

I'll review any moment soon

**Comment by Aline160** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/language-selector/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+	languageBtn.addEventListener('click', () => {
+		languageBtnIcons.forEach((cur_languageBtnIcon) => {
+			cur_languageBtnIcon.classList.toggle('d-none');
+		});
+	});
+</script>
+
+<style>
+.portlet {
```
[the remark this answers is not in the document]
Comment (kylembischof) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/language-selector/ddm-template.ftl:73]: can you also make sure to sort classes and attributes in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<div class="account position-relative" tabindex="9">
+<button class="account-info align-items-center d-flex ${themeDisplay.isSignedIn()?then('py-1 pl-2', '')}" data-toggle="liferay-dropdown" id="account-menu-id">
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/account-menu/index.html:2]: Some of the indents are off in this file. This first `<button>` does not line up with the `</button>` tag. There are some other indents that don't line up in this file too, can you find them?

Make sure the indenting is accurate in files to have better readability for everyone working on it in the future

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/sites/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<div class="f-navigation-menu-item">
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/sites/index.html:1]: can the indents in this file be double checked too?

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by Mylena-Rodrigues** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Good job guys!

**Comment by kevenleone** [comment]
ci:forward

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:forward

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by samuelleolopes** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**